code/components/voip-server-mumble/src/client.cpp

```diff
@@ -993,7 +995,8 @@ int Client_voiceMsg(client_t *client, uint8_t *data, int len)
 			buffer[0] = (uint8_t) (type | 2);
 			Log_debug("Whisper session %d", vt->sessions[i]);
 			while (Client_iterate(&c) != NULL) {
-				if (c->sessionId == vt->sessions[i]) {
+				channel_t* ch = (channel_t*)client->channel;
+				if (!targeted_channels[ch->id] && c->sessionId == vt->sessions[i]) {
```

Comment:
`.find(ch->id) != .end()` is preferred here (non-mutating, your use of `operator[]` will append the current channel with default value - false - on checking), in fact since you're not ever using false _intentionally_ you could also use a `set` - or rather, an `eastl::fixed_set`.